src/pages/index.astro

```diff
@@ -0,0 +1,542 @@
+              {category.alternatives && (
+                <div class="alternatives">
+                  <h4 class="alternatives-title">Alternatives:</h4>
+                  <ul class="alternatives-list">
+                    {Object.entries(category.alternatives).map(
+                      ([alternativeName, alternativeUrl]) => (
+                        <li>
+                          <a
+                            href={alternativeUrl}
+                            target="_blank"
+                            rel="noopener"
+                            class="alternative-link"
+                          >
+                            {alternativeName}
+                          </a>
+                        </li>
+                      )
+                    )}
+                  </ul>
+                </div>
+              )}
+              <div class="comparisons">
+                {Object.entries(category.comparisons).map(
+                  ([comparisonName, comparison]) => (
+                    <div class="comparison" id={comparisonName}>
+                      <h3 class="comparison-title">
+                        <a href={`#${comparisonName}`}>
+                          {titleCase(comparisonName.replaceAll('_', ' '))}
+                        </a>
+                      </h3>
+                      {comparison.alternatives && (
+                        <div class="alternatives">
+                          <h4 class="alternatives-title">Alternatives:</h4>
+                          <ul class="alternatives-list">
+                            {Object.entries(comparison.alternatives).map(
+                              ([alternativeName, alternativeUrl]) => (
+                                <li>
+                                  <a
+                                    href={alternativeUrl}
+                                    target="_blank"
+                                    rel="noopener"
+                                    class="alternative-link"
+                                  >
+                                    {alternativeName}
+                                  </a>
+                                </li>
+                              )
+                            )}
+                          </ul>
+                        </div>
+                      )}
+                      <div class="engines">
+                        {Object.entries(comparison.engines).map(
+                          ([engineName, code]) => (
+                            <>
+                              <p>{engineName}</p>
+                              <Prism lang="js" code={code} />
+                            </>
+                          )
+                        )}
+                      </div>
+                    </div>
+                  )
+                )}
+              </div>
+            </section>
+          ))
+        }
+      </div>
+    </main>
+
+    <footer>
+      <p>Made by some Engineer Gamers</p>
+    </footer>
```

Comment:
Removing the credit to Adam and Zack and Hubspot also seems out of scope for the PR, unless Adam and Zack have endorsed this.

Reply:
Yes sorry, they should still be credited. My work never had any merit for a change which is why I upgraded the message to credit a pseudonym as a joke. I still want to include that to credit community contribution but the names of the original creators shall still exist as the only proper nouns in the statement.